### DISCUSSION TITLE
A very silly Question: How to remove the --> /bin/zsh in Prompt?

### DISCUSSION THREAD

**Discussion by hhlp** [starship/starship#1936] [Q&A] [answered]
user: hhlp on hhlp ~ with /bin/zsh

The question say it all..... 

What module provide this function?

starship 0.46.2

´´´ 
add_newline = false

format = """\
$username\
$hostname\
$kubernetes\
$directory\
$git_branch\
$git_commit\
$git_state\
$git_status\
$docker_context\
$package\
$cmake\
$golang\
$java\
$julia\
$nodejs\
$perl\
$php\
$python\
$ruby\
$rust\
$nix_shell\
$aws\
$env_var\
$cmd_duration\
$line_break\
$jobs\
$character"""

scan_timeout = 10

´´´ 

Regards.,

**Comment by andytom** [comment, marked as answer]
Hi @hhlp, if you run the command `starship explain` it should display a breakdown of your prompt showing which modules produces each section.
